web/public/locales/en/common.json

```diff
@@ -246,7 +247,7 @@
     "firstPost": "Hi! I created this _helpbutton_ at {0}, leave a comment if you want",
     "share": "Share",
     "copy": "Copy Link",
-    "delete": "Finalize",
+    "delete": "Delete",
```

Comment:
why are you changing back to delete? if previously you said should be finalize?

Reply:
Made some test and it was confusing for some users. We need to find an appropiate term, but for now delete seems the best,

Reply:
did you check the term in other languages?

Reply:
no, I was going to make an overall check of the translations after we do this push. Although i can switch it in spanish